### PR TITLE
Update from Weblate.

### PR DESCRIPTION
**Pull request by xwikiorgci**

### CODE REVIEW

File: application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarTranslations.fr.xml [hunk 1]
```diff
@@ -58,14 +58,14 @@ MoccaCalendar.calendar.create=Créer un nouveau calendrier
...
-MoccaCalendar.calendar.plainlist.noupcomingevents=Aucun événement à venir trouvé
+MoccaCalendar.calendar.plainlist.nopastevents=Aucun événement trouvé.
+MoccaCalendar.calendar.plainlist.noupcomingevents=Aucun événement à venir trouvé.
 MoccaCalendar.calendar.plainlist.titlepast=Evènements passés
 MoccaCalendar.calendar.plainlist.titleupcoming=Événements à venir
 MoccaCalendar.calendar.title=Calendrier
 MoccaCalendar.calendar.viewall=Vue d'ensemble de tous les calendriers
-MoccaCalendar.calendarevent.backlink=Retour au calendier &amp;\#171;{0}&amp;\#187;
-MoccaCalendar.calendarevent.confirmDelete=Voulez-vous vraiment supprimer l''évènement &amp;\#171;{0}&amp;\#187; ? 
+MoccaCalendar.calendarevent.backlink=Retour au calendier &amp;#171;{0}&amp;#187;
```
Comment (vmassol) [inline application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarTranslations.fr.xml]: s/calendier/calendrier ;)

Comment (ClemensRobbenhaar) [reply application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarTranslations.fr.xml]: I was actually more worried about the missing escapes for the `#` signs and did not merge the pull request; but now I tested it and it does not cause any problems.

**Comment by ClemensRobbenhaar** [comment]
Thanks for the pull request!

File: application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarTranslations.fr.xml (hunk 1 above)
Comment (vmassol) [reply application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarTranslations.fr.xml]: I've fixed "calendier" on weblate

**Pull request merged**